### PR TITLE
feat:  Rename qrCode wording to message name in API endpoints, requests and response

### PR DESCRIPTION
**Pull request by x1m3**

### CODE REVIEW

File: api/api.yaml [hunk 1]
```diff
@@ -101,11 +101,11 @@ paths:
...
 
-  /v2/{identifier}/authentication/qrcode:
+  /v2/{identifier}/authentication:
     post:
-      summary: Get Authentication QRCode
-      operationId: authQRCode
-      description: This endpoint returns a QR code to authenticate, when scanned it a callback is made to the callback endpoint.
+      summary: Get Authentication Message
+      operationId: Authentication
+      description: This endpoint returns an authentication messag
```
Comment (amonsosanz) [inline api/api.yaml:108]: message

File: api/api.yaml [hunk 2]
```diff
@@ -861,11 +859,11 @@ paths:
...
 
-  /v2/identities/{identifier}/credentials/{id}/qrcode:
+  /v2/identities/{identifier}/credentials/{id}/offer:
     get:
-      summary: Get Credentials QR code
-      operationId: GetCredentialQrCode
-      description: Returns a a json that can be used to create the QR Code to scan for accepting a claim. |
+      summary: Get Credentials Offer
+      operationId: GetCredentialOffer
+      description: Returns universal/deep link to offer or raw offer message can be used for accepting a claim. |
```
Comment (amonsosanz) [inline api/api.yaml:866]: Returns **a** universal/deep-link to **an** offer or raw offer message **that** can be used for accepting a claim.

File: api/api.yaml [hunk 3]
```diff
@@ -883,16 +881,16 @@ paths:
             enum: [ universalLink, deepLink, raw ]
           description: >
             Type:
-              * `universalLink` - (default value) Returns a deeplink. The preferred and more standard way to access the QR  
-              * `deepLink` -  Returns a QR code with a link redirection to the raw content.   
-              * `raw` - Returns the raw QR code.
+              * `universalLink` - (default value) Returns a deeplink. The preferred and more standard way to access the offer message
```
Comment (amonsosanz) [inline api/api.yaml:884]: Returns a universal link

File: api/api.yaml [hunk 4]
```diff
@@ -1240,19 +1238,19 @@ paths:
...
 
-  /v2/identities/{identifier}/credentials/links/{id}/qrcode:
+  /v2/identities/{identifier}/credentials/links/{id}/offer:
     post:
-      summary: Create Link QR Code
-      operationId: CreateLinkQrCode
-      description: Create a QR code for the provided link
+      summary: Create a credential offer for a link
+      operationId: CreateLinkOffer
+      description: Create a credential offer suitable to be used as qrCode for the provided link
```
Comment (amonsosanz) [inline api/api.yaml:1245]: shouldn't we remove `qrCode` here?

**Review by amonsosanz — APPROVED** [review]

**Review by martinsaporiti — APPROVED** [review]

**Pull request merged**